upgrade_test.py

```diff
@@ -439,6 +442,29 @@ def fill_and_verify_db_data(self, note, pre_fill=False, rewrite_data=True):
             self.log.info('Re-Populate DB with many types of tables and data')
             self.fill_db_data()
 
+    def jepsen_setup(self):
+        loader_node = self.loaders.nodes[0]
+        if not loader_node.distro.is_debian10:
+            raise Exception('Jepsen can only run on Debian 10')
+        db_node = self.db_cluster.nodes[0]
+        remoter = loder_node.remoter
+        remoter.sudo("apt-get install -y libjna-java gnuplot graphviz git")
+        remoter.run(shell_script_cmd(f"""\
+            curl -O https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein
+            chmod +x lein
+            ./lein
+            git clone {self.params.get('jepsen_scylla_repo')} jepsen-scylla
+        """))
+        for db_node in self.db_cluster.nodes:
+            remoter.run(f"ssh-keyscan -t rsa {db_node.ip_address} >> ~/.ssh/known_hosts")
+        remoter.send_files(os.path.expanduser(self.db_cluster.nodes[0].ssh_login_info["key_file"]), DB_SSH_KEY)
+
+    def jepsen_test(self):
+        loader_node = self.loaders.nodes[0]
+        nodes = " ".join(f"--node {node.ip_address}" for node in self.db_cluster.nodes)
+        creds = f"--username {self.db_cluster.nodes[0].ssh_login_info['user']} --ssh-private-key ~/{DB_SSH_KEY}"
+        loader_node.remoter.run(f'cd ~/jepsen-scylla && ~/lein run test -w list-append --time-limit 200 --concurrency 2n -r 2 {nodes} {creds} --no-install-scylla'
+
```

Comment:
We need to think how to reuse code from `jepsen_test.py` here.

Reply:
I moved the general setup code to sdcm/utils/jepsen.py